.bit/responses/1.3-Week 1 Step 3.md

```diff
@@ -10,12 +10,12 @@ name: Week 1 Step 3
 Week 1 Step 3 ⬤⬤⬤◯◯◯◯◯◯ | 🕐 Estimated completion: 10-15 minutes
 
 # Getting Cat Pics with Postman 🐱
-*In this step, you will be using Postman to test the [CataaS API](https://cataas.com/#/) by sending a GET request.*
+*In this step, you will be using Postman Desktop Application to test the [CataaS API](https://cataas.com/#/) by sending a GET request.*
 
 ## ✅  Tasks:
-- [ ] ***1:*** Read and explore the documentation for the [Cataas API](https://cataas.com/#/) 
-- [ ] ***2:*** With Postman, specify the API Endpoint and set the parameters to send a GET request to the [CataaS API](https://cataas.com/#/) and receive a cat picture with "Serverless" written on it 
-- [ ] 🚀 Comment the picture you receive from the API in this issue/pull request to move on 
+- [ ] ***1:*** Read and explore the documentation for the [Cataas API](https://cataas.com/#/)
+- [ ] ***2:*** With Postman, specify the API Endpoint and set the parameters to send a GET request to the [CataaS API](https://cataas.com/#/) and receive a cat picture with "Serverless" written on it
+- [ ] 🚀 Comment the picture you receive from the API in this issue/pull request to move on
```

Comment:
Comment the picture?

Reply:
I think what is meant. _Please respond by posting a comment. Within the comment attach the image you received from the API in this issue/pull request to move on_ 

Let me know if that si clearer to you and ill update that language

Reply:
@creaturenex Maybe be more concise by saying: `On this issue or pull request, please post a comment with the cat picture received from the API.`

Reply:
Sorry for the late reply, count of context `Comment the picture` seemed odd,  I agree with @emsesc  I think her suggestion makes it clearer. I initially thought it meant write "I have a picture of a cat" 🤣